<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DragGAN</class>
 <widget class="QMainWindow" name="DragGAN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="7,10,0" columnstretch="1,3">
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="1" rowspan="3">
     <widget class="ImageWidget" name="Image_Widget" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="model" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>14</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0,0,0" columnstretch="2,0,0,0,0,0,0">
       <item row="2" column="3" colspan="4">
        <widget class="QPushButton" name="Browse_PushButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="Minus4Seed_PushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QPushButton" name="Recent_PushButton">
         <property name="text">
          <string>Recent...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="Wp_CheckBox">
         <property name="text">
          <string>W+</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1" colspan="6">
        <widget class="QComboBox" name="Device_ComboBox"/>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="W_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>W</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="Generate_PushButton">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Seed_LineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="6">
        <widget class="QLineEdit" name="Pickle_LineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Device_Label">
         <property name="text">
          <string>Device</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Latent_Label">
         <property name="text">
          <string>Latent</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="Plus4Seed_PushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Pickle_Label">
         <property name="text">
          <string>Pickle</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="Seed_Label">
         <property name="text">
          <string>Seed</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="RandomSeed_CheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Random Seed</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="SaveReal_PushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="drag" native="true">
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="4,4,1,1,4">
       <item row="0" column="1" colspan="3">
        <widget class="QPushButton" name="AddPoint_PushButton">
         <property name="text">
          <string>Add point</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Drag_Label">
         <property name="text">
          <string>Drag</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="StepSize_LineEdit"/>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="Stop_PushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="11" column="4">
        <widget class="QLabel" name="Drag_Label_5">
         <property name="text">
          <string>Lambda</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="ResetPoint_PushButton">
         <property name="text">
          <string>Reset point</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QPushButton" name="Minus4Lambda_PushButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="Steps_Label">
         <property name="text">
          <string>Steps:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="R1_LineEdit"/>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QPushButton" name="Start_PushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="FixedArea_PushButton">
         <property name="text">
          <string>Fixed area</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="StepSize_Label">
         <property name="text">
          <string>Step Size</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="Mask_Label">
         <property name="text">
          <string>Mask</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="3">
        <widget class="QCheckBox" name="ShowMask_CheckBox">
         <property name="text">
          <string>Show mask</string>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="QLabel" name="Drag_Label_4">
         <property name="text">
          <string>Radius</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="Reset4R1_PushButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="R1_Label">
         <property name="text">
          <string>R1</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QPushButton" name="Plus4Lambda_PushButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="Reset4StepSize_PushButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="Lambda_LineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>0.5</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QPushButton" name="ResetMask_PushButton">
         <property name="text">
          <string>Reset mask</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="Radius_LineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QPushButton" name="FlexibleArea_PushButton">
         <property name="text">
          <string>Flexible area</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="Plus4Radius_PushButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QPushButton" name="Minus4Radius_PushButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QPushButton" name="Reset4R2_PushButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="R2_Label">
         <property name="text">
          <string>R2</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="R2_LineEdit"/>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="StepNumber_Label">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="SaveGenerate_PushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">components.ImageWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
